### ISSUE TITLE
Link is not dead but errors have been found

### ISSUE THREAD

**Issue by seriousmac** [sindresorhus/awesome-lint#50]
awesome-lint finds errors following 2 cases which is not dead

i) Link to https://arxiv.org/abs/* is dead
ii) Link to https://arxiv.org/pdf/* is dead

For example, Link to https://arxiv.org/pdf/1611.01603.pdf is dead

**Comment by automata** [comment]
Getting the same for https://vimeo.com/163522126:

<img width="614" alt="screen shot 2018-12-02 at 09 58 03" src="https://user-images.githubusercontent.com/49062/49339445-d6e85b00-f618-11e8-8fcf-f08f2648b33b.png">

**Comment by sindresorhus** [comment]
Please report these false-positives to https://github.com/transitive-bullshit/check-links It's the package used to check the links.

**Comment by HorlogeSkynet** [comment]
So I guess it would need to make https://github.com/transitive-bullshit/check-links/blob/3b08216d01a1107f0c692f611b11274f996006ca/lib/check-link.js#L38 customizable, and increase it in this module I guess ?

(cc @transitive-bullshit)

**Comment by transitive-bullshit** [comment]
I'll have some time to look into this soon. For now, I've opened an [issue](https://github.com/transitive-bullshit/check-links/issues/4) to track this.

**Comment by transitive-bullshit** [comment]
@HorlogeSkynet can you provide some additional false negatives in the above issue that you're seeing problems with?

**Comment by gwaygenomics** [comment]
Similar false positive error found by @CompSciLauren in https://github.com/sindresorhus/awesome/pull/1595#issuecomment-516200153

Oddly, when I ran `awesome-lint` personally, I did not see this error

```bash
(base) wm962-fdf:awesome-cytodata gway$ awesome-lint
⠙ Linting(node:6962) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
✖ Linting

  readme.md:1:1
  ✖  1:1  The repository should have "awesome" as a GitHub topic       remark-lint:awesome/github
  ✖  1:1  The repository should have "awesome-list" as a GitHub topic  remark-lint:awesome/github

  2 errors
```

(These are also false postives, probably b/c I am running awesome-lint on my fork)

**Comment by transitive-bullshit** [comment]
There is definitely a broader issue here with the remark-lint:no-dead-urls rule. See my proposal in #89 to work around it.
